### PR TITLE
update whl for multi lang rec

### PR DESCRIPTION
**Pull request by andyjpaddle**
update whl for multi lang rec

### CODE REVIEW

File: paddleocr.py [hunk 1]
```diff
@@ -79,6 +83,66 @@
...
+                    'dict_path': './ppocr/utils/dict/ka_dict.txt'
+                },
+                'latin': {
+                    'url':
+                    'https://paddleocr.bj.bcebos.com/PP-OCRv3/multilingual/latin_ppocr_PP-OCRv3_rec_infer.tar',
+                    'dict_path': './ppocr/utils/dict/latin_dict.txt'
+                },
+                'arabic': {
+                    'url':
+                    'https://paddleocr.bj.bcebos.com/PP-OCRv3/multilingual/arabic_ppocr_PP-OCRv3_rec_infer.tar',
```
Comment (tink2123) [inline paddleocr.py:133]: `arabic_ppocr_PP-OCRv3_rec_infer.tar` --> `arabic_PP-OCRv3_rec_infer.tar`

File: paddleocr.py [hunk 2]
```diff
@@ -79,6 +83,66 @@
...
+                    'dict_path': './ppocr/utils/dict/te_dict.txt'
+                },
+                'ka': {
+                    'url':
+                    'https://paddleocr.bj.bcebos.com/PP-OCRv3/multilingual/ka_PP-OCRv3_rec_infer.tar',
+                    'dict_path': './ppocr/utils/dict/ka_dict.txt'
+                },
+                'latin': {
+                    'url':
+                    'https://paddleocr.bj.bcebos.com/PP-OCRv3/multilingual/latin_ppocr_PP-OCRv3_rec_infer.tar',
```
Comment (tink2123) [inline paddleocr.py:128]: 地址修改，同上

File: paddleocr.py [hunk 3]
```diff
@@ -79,6 +83,66 @@
...
+                    'dict_path': './ppocr/utils/dict/latin_dict.txt'
+                },
+                'arabic': {
+                    'url':
+                    'https://paddleocr.bj.bcebos.com/PP-OCRv3/multilingual/arabic_ppocr_PP-OCRv3_rec_infer.tar',
+                    'dict_path': './ppocr/utils/dict/arabic_dict.txt'
+                },
+                'cyrillic': {
+                    'url':
+                    'https://paddleocr.bj.bcebos.com/PP-OCRv3/multilingual/cyrillic_ppocr_PP-OCRv3_rec_infer.tar',
```
Comment (tink2123) [inline paddleocr.py:138]: `cyrillic_ppocr_PP-OCRv3_rec_infer.tar` --> `cyrillic_PP-OCRv3_rec_infer.tar`

File: paddleocr.py [hunk 4]
```diff
@@ -79,6 +83,66 @@
...
+                    'dict_path': './ppocr/utils/dict/cyrillic_dict.txt'
+                },
+                'devanagari': {
+                    'url':
+                    'https://paddleocr.bj.bcebos.com/PP-OCRv3/multilingual/devanagari_ppocr_PP-OCRv3_rec_infer.tar',
```
Comment (tink2123) [inline paddleocr.py:143]: `devanagari_ppocr_PP-OCRv3_rec_infer.tar` --> `devanagari_PP-OCRv3_rec_infer.tar`

File: paddleocr.py [hunk 5]
```diff
@@ -285,6 +349,8 @@ def parse_lang(lang):
         det_lang = "ch"
     elif lang == 'structure':
         det_lang = 'structure'
+    elif lang == "ml":
```
Comment (tink2123) [inline paddleocr.py:352]: 这个判断其实是没有生效的，可以改成：
```
    if lang == "ch":
        det_lang = "ch"
    elif lang == 'structure':
        det_lang = 'structure'
    elif lang in ["en", "latin"]:
        det_lang = 'en'
    else:
        det_lang = "ml"
```

File: paddleocr.py [hunk 6]
```diff
@@ -79,6 +83,66 @@
                     'https://paddleocr.bj.bcebos.com/PP-OCRv3/english/en_PP-OCRv3_rec_infer.tar',
                     'dict_path': './ppocr/utils/en_dict.txt'
                 },
+                'french': {
+                    'url':
+                    'https://paddleocr.bj.bcebos.com/PP-OCRv3/multilingual/latin_ppocr_PP-OCRv3_rec_infer.tar',
```
[the remark this answers is not in the document]
Comment (andyjpaddle) [reply paddleocr.py]: done,ths

File: paddleocr.py (hunk 1 above)
Comment (andyjpaddle) [reply paddleocr.py]: done,ths

File: paddleocr.py (hunk 2 above)
Comment (andyjpaddle) [reply paddleocr.py]: done, ths

File: paddleocr.py (hunk 3 above)
Comment (andyjpaddle) [reply paddleocr.py]: done,ths

File: paddleocr.py (hunk 4 above)
Comment (andyjpaddle) [reply paddleocr.py]: done, ths

File: paddleocr.py (hunk 5 above)
Comment (andyjpaddle) [reply paddleocr.py]: done,ths

**Review by tink2123 — APPROVED** [review]
LGTM

**Pull request merged**